### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by luomai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,15 +45,27 @@ rm *.so
 
 ## Training your model
 
-Runs `train.py`, it will automatically download MSCOCO 2017 dataset into `dataset/coco17`.
+Training the model is implemented using TensorFlow. To run `train.py`, you would need the following packages 
+in your virtual environment (Python <=3.6):
+
+```bash
+pip install tensorflow
+pip install tensorlayer
```
Comment (lgarithm) [inline README.md]: you can use `pip install -r requirements.txt`

**Comment by luomai** [comment]
Please approve this PR. @lgarithm

File: requirements.txt [hunk 2]
```diff
@@ -3,5 +3,5 @@ numpy
 easydict
 opencv-python
 tensorflow
-nltk # required by tensorlayer
-# pycocotools  # must be installed after cython and numpy are installed
+tensorlayer
+pycocotools  # must be installed after cython and numpy are installed
```
Comment (lgarithm) [inline requirements.txt]: pycocotools must be installed after cython and numpy are installed, as commented.

Comment (luomai) [reply requirements.txt]: does pip install follow the order in the requirements.txt?

**Pull request merged**